python/mxnet/recordio.py

```diff
@@ -132,31 +198,75 @@ def open(self):
                 self.keys.append(key)
 
     def close(self):
+        """Closes the record file."""
         if not self.is_open:
             return
         super(MXIndexedRecordIO, self).close()
         self.fidx.close()
 
     def seek(self, idx):
-        """Query current read head position."""
+        """Sets the current read pointer position.
+
+        This function is internally called by `read_idx(idx)` to find the current
+        reader pointer position. It doesn't return anything."""
         assert not self.writable
         pos = ctypes.c_size_t(self.idx[idx])
         check_call(_LIB.MXRecordIOReaderSeek(self.handle, pos))
 
     def tell(self):
-        """Query current write head position."""
+        """Returns the current position of write head.
+
+        Example usage:
+        ----------
+        >>> record = mx.recordio.MXIndexedRecordIO('tmp.idx', 'tmp.rec', 'w')
+        >>> print(record.tell())
+        0
+        >>> for i in range(5):
+        ...     record.write_idx(i, 'record_%d'%i)
+        ...     print(record.tell())
+        16
+        32
+        48
+        64
+        80
+        """
         assert self.writable
         pos = ctypes.c_size_t()
         check_call(_LIB.MXRecordIOWriterTell(self.handle, ctypes.byref(pos)))
         return pos.value
 
     def read_idx(self, idx):
-        """Read record with index."""
+        """Returns a record at given index.
```

Comment:
Perhaps change "a" to "the" since there is only one record at the given index. Is that right?